sphinx_gallery/gen_gallery.py

```diff
@@ -186,7 +186,7 @@ def _complete_gallery_conf(sphinx_gallery_conf, src_dir, plot_gallery,
 
     # Ensure the first cell text is a string if we have it
     first_cell = gallery_conf.get("first_notebook_cell")
-    if not isinstance(first_cell, basestring):
+    if not isinstance(first_cell, (basestring, type(None))):
         raise ValueError("The 'first_notebook_cell' parameter must be type str"
```

Comment:
should say "str or None" now?

Reply:
do these need to be in two different parts? e.g. `isinstance(` followed by `is None`?

Reply:
I usually do `isinsance(x, y) and x is not None` but what you have here is fine. My comment is about the text in the error to the user, `None` should also be offered as an option

Reply:
Still need to say this can be `str or None` in the error message :)

Pushed a commit for this